pkg/logger/log.go

```diff
@@ -138,33 +112,43 @@ func (e LogStatus) Color() *color.Color {
 }
 
 type Logger interface {
-	Emit(LogStatus, string, ...interface{})
+	Emit(LogStatus, string, ...any)
```

Comment:
Not really sure how much Go needs yet another logger XD

It's a fun exercise, as long as you recognize it as being a time sink.